src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -135,6 +157,13 @@ export default class NewDataset extends Vue {
     T: null,
     C: null
   };
+  strides: { [dimension: string]: number | string | null } = {
+    XY: null,
+    Z: null,
+    T: null,
+    C: null
+  };
+  areStridesSetFromFile = false;
```

Comment:
missing a type definition?

Reply:
It could be explicitly defined as `boolean` but setting it to `false` implicitly defines it as `boolean`.
Should I edit the declaration?

Reply:
I think it is better to define types!